web/file_manager_languages.php

```diff
@@ -0,0 +1,124 @@
+<?php
+/**
+ * !Warning!
+ * this file must be included, not required.
+ */
+if (!isset($dist_config)) {
+	require_once "index.php";
+	exit();
+}
+/**
+ * get_language_from_system()
+ *
+ * isolate the hestiacp commands from this file
+ *
+ * @return string the language code
+ */
+function get_language_from_system() {
+	include $_SERVER["DOCUMENT_ROOT"] . "/inc/main.php";
+	/**
+	 * get data
+	 */
+	exec(HESTIA_CMD . "v-list-sys-languages json", $output, $return_var);
+	$languages = json_decode(implode("", $output), true);
+	return in_array($_SESSION["language"], $languages) ? $_SESSION["language"] : "en";
+}
+
+/**
+ * Language Switch for filegator
+ *
+ * @see https://docs.filegator.io/translations/default.html#rtl-support
+ */
+switch (get_language_from_system()) {
```

Comment:
SESSION['language']  is fine we should prefer the user language above server language...